feat(changelog): domains-and-dns-changed-increase-in-the-cost-of-the-fr- 2024-05-15 (#3171)
scaleway/docs-content: changelog/may2024/2024-05-15-domains-and-dns-changed-increase-in-the-cost-of-the-fr-.mdx

---

> title: Increase in the cost of the .FR extension

Reviewer: ```suggestion
title: Increase in the cost of the .FR extension renewal
```

status: changed
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer 2: ```suggestion
    fullname: 'Join the #domains channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-05-15
category: network
product: domains-and-dns
---

> As recently announced by AFNIC, the price of the .FR extension registration fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price adjustment [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).

Reviewer: ```suggestion
As recently announced by AFNIC, the price of the .FR extension renewal fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price adjustment [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).
```

> As you may see, the AFNIC announced that the .FR extension will increase its price by 11,1% on 1st March 2024 : https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/
> 
> This augmentation will also impact the cost of .FR domains you buy from Scaleway, starting June 1st 2024.

Reviewer 3: ```suggestion
Certainly! Here's a revised version:

As recently announced by AFNIC, the price of the .FR extension registration fee is about to increase by 11.1% effective from March 1st, 2024. You can find further details on this price increase [here](https://www.afnic.fr/observatoire-ressources/actualites/le-fr-passe-de-4-56e-a-507e-a-partir-du-1er-mars-2024/).

This price adjustment will subsequently influence the pricing of .FR domains purchased through Scaleway, commencing from June 1st, 2024.
```
